docker/Dockerfile.BuildJekyll

```diff
@@ -0,0 +1,21 @@
+FROM ruby@sha256:c74394d0a4a05a3068ca9d0c53c36d451e27773264b568ae16ae24be9ad5621d
+# ruby:3.0.0
```

Comment:
No shade if you did intend it this way, but if you're using buildkit (happy to help you get that updated otherwise) this should be:

```suggestion
FROM ruby:3.0.0@sha256:c74394d0a4a05a3068ca9d0c53c36d451e27773264b568ae16ae24be9ad5621d
```